app/models/admin_post.rb

```diff
@@ -76,4 +77,21 @@ def expire_cached_home_admin_posts
       Rails.cache.delete("home/index/home_admin_posts")
     end
   end
+
+  def inherit_translated_post_comment_permissions
+    return unless translated_post_id.present? && AdminPost.find_by(id: translated_post_id)
```

Comment:
Can this be `return if translated_post.blank?`?

Reply:
Yeah, I was clearly overthinking here -- I wanted to make sure things wouldn't explode if we were calling this on a post with a translated post that didn't exist in the database. But the translated_post_id validation runs before we save, so we never actually get to the point of invoking this on a post without a valid translated_post_id.